CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ All notable changes to this project will be documented in this file.
 #### Removed
 
 #### Fixed
+- Default Swift version now correctly set to 4.2.
```

Comment:
How about: `- Issue where Swift version was incorrectly set to 5.0 when it should have been 4.2.`?